App/Components/ListItem.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react'
+import { View, Text, TouchableOpacity } from 'react-native'
+
+import * as s from '../Themes/Constants'
+
+interface Props {
+  id: string
+  title: string
+  subtitle?: string
+  renderLeftItem?: () => JSX.Element
+  renderRightItems?: () => JSX.Element[]
+  onPress?: (id: string) => void
+}
+
+class ListItem extends React.PureComponent<Props> {
```

Comment:
New general use `ListItem`. We can use this many times we need a list item in a `FlatList`